src/prx/rinex_obs/test/benchmark.py

```diff
@@ -61,7 +59,6 @@ def generate_data():
     for case in cases:
         for parser in [
             ("prx", prx_obs_parse),
-            ("georinex", helpers.parse_rinex_obs_file),
```

Comment:
Makes the benchmark much faster, and we know that prx's parser is faster, so no need to compare to georinex's anymore.